sdk/keyvault/azkeys/client.go

```diff
@@ -1508,7 +1427,7 @@ func updateKeyRotationPolicyResponseFromGenerated(i generated.KeyVaultClientUpda
 		attribs = &KeyRotationPolicyAttributes{
 			ExpiryTime: i.Attributes.ExpiryTime,
 			Created:    i.Attributes.Created,
```

Comment:
Same here (I won't mention it again for brevity, so please check for all occurrences).